spec/factories/case/offender_sars.rb

```diff
@@ -35,6 +35,9 @@
     recipient                       { 'subject_recipient' }
     third_party                     { false }
     flag_as_high_profile            { false }
+    num_exempt_pages                { '200' }
```

Comment:
Think this needs to be `number_*` rather than `num_*` 👍

Reply:
These attributes have been removed from the factory.